e2e_tests/test_standalone/test_study_list.py

```diff
@@ -20,3 +23,46 @@ def test_home(
     title = element.text_content()
     assert title is not None
     assert title == "Optuna Dashboard (Wasm ver.)"
+
+
+def test_load_storage(
+    page: Page,
+    server_url: str,
+) -> None:
+    current_dir = os.getcwd()
```

Comment:
How about using `tempfile.TemporaryDirectory` to avoid the SQLite3 file from being left when running the E2E tests on our local machines?